ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -592,6 +630,7 @@ export default class AdvancedTab extends PureComponent {
         {this.renderAutoLockTimeLimit()}
         {this.renderUserDataBackup()}
         {this.renderDismissSeedBackupReminderControl()}
+        {this.renderOverrideContentSecurityPolicyHeader()}
```

Comment:
Should this option should only be displayed for FireFox users?

Reply:
Yes, firefox only makes sense